specification/relay/resource-manager/Microsoft.Relay/stable/2016-07-01/relay.json

```diff
@@ -537,6 +578,47 @@
             }
           }
         }
+      },
+      "post": {
```

Comment:
what does the POST on authorizationRule do? Is it creating or updating it? It should be done only via a PUT

Reply:
it retrieves the AuthorizationRule details.

Reply:
Yes. In this API version, POST on the AuthorizationRules URL functions like a GET but does return the keys associated with the Rule.  Whereas GET doesn't return keys.  In future API versions 2015 we use the RegenerateKeys, ListKeys actions to get and manipulate keys to align better with other RPs.